src/wp-includes/rest-api/endpoints/class-wp-rest-template-autosaves-controller.php

```diff
@@ -0,0 +1,276 @@
+				),
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_items' ),
+					'permission_callback' => array( $this, 'get_items_permissions_check' ),
+					'args'                => $this->get_collection_params(),
+				),
+				array(
+					'methods'             => WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'create_item' ),
+					'permission_callback' => array( $this, 'create_item_permissions_check' ),
+					'args'                => $this->parent_controller->get_endpoint_args_for_item_schema( WP_REST_Server::EDITABLE ),
+				),
+				'schema' => array( $this, 'get_public_item_schema' ),
+			)
+		);
+
+		register_rest_route(
+			$this->namespace,
+			sprintf(
+				'/%s/(?P<parent>%s%s)/%s/%s',
+				$this->parent_base,
+				/*
+				 * Matches theme's directory: `/themes/<subdirectory>/<theme>/` or `/themes/<theme>/`.
+				 * Excludes invalid directory name characters: `/:<>*?"|`.
+				 */
+				'([^\/:<>\*\?"\|]+(?:\/[^\/:<>\*\?"\|]+)?)',
+				// Matches the template name.
+				'[\/\w%-]+',
+				$this->rest_base,
+				'(?P<id>[\d]+)'
+			),
+			array(
+				'args'   => array(
+					'parent' => array(
+						'description'       => __( 'The id of a template' ),
+						'type'              => 'string',
+						'sanitize_callback' => array( $this->parent_controller, '_sanitize_template_id' ),
+					),
+					'id'     => array(
+						'description' => __( 'The ID for the autosave.' ),
+						'type'        => 'integer',
+					),
+				),
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_item' ),
+					'permission_callback' => array( $this->revisions_controller, 'get_item_permissions_check' ),
+					'args'                => array(
+						'context' => $this->get_context_param( array( 'default' => 'view' ) ),
+					),
+				),
+				'schema' => array( $this, 'get_public_item_schema' ),
+			)
+		);
+	}
+
+	/**
+	 * Prepares the item for the REST response.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param mixed           $item    WordPress representation of the item.
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response Response object.
+	 */
+	public function prepare_item_for_response( $item, $request ) {
+		$template = _build_block_template_result_from_post( $item );
+		$response = $this->parent_controller->prepare_item_for_response( $template, $request );
+
+		$fields = $this->get_fields_for_response( $request );
+		$data   = $response->get_data();
+
+		if ( in_array( 'parent', $fields, true ) ) {
+			$data['parent'] = (int) $item->post_parent;
+		}
+
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
+		$data    = $this->filter_response_by_context( $data, $context );
+
+		// Wrap the data in a response object.
+		$response = new WP_REST_Response( $data );
+
+		if ( rest_is_field_included( '_links', $fields ) || rest_is_field_included( '_embedded', $fields ) ) {
```

Comment:
I assume templates don't support meta?

Reply:
Not official. But it does support it like this . 

https://github.com/WordPress/wordpress-develop/blob/9b2ed59437441459adae547cac32b3a31d1a9d30/src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php#L585-L592